static/bonescript.js

```diff
@@ -24,12 +15,24 @@ function setTargetAddress(address, handlers) {
         script.type = 'text/javascript';
         script.src = url;
         script.charset = 'UTF-8';
+        script.onerror = function() {
```

Comment:
Sorry, I didn't understand the first point. The script will be appended to
HTML document, right?

Yes, Actually first I wrote synchronous code, and then I realized this
would not work.
Later, I pushed another commit of asynchronous code. And sure, I will keep
in mind about "squashing" the commits next time.

On Wed, Mar 23, 2016 at 5:56 PM, Jason Kridner <notifications@github.com>
wrote:

> In static/bonescript.js
> <https://github.com/jadonk/bone101/pull/34#discussion_r57150448>:
>
> > @@ -24,12 +15,24 @@ function setTargetAddress(address, handlers) {
> >          script.type = 'text/javascript';
> >          script.src = url;
> >          script.charset = 'UTF-8';
> > +        script.onerror = function() {
>
> Why are you putting this in the HTML document context? The local namespace
> is perfectly fine to use without reaching into the HTML document.
>
> Glad you figured out that your synchronous logic wouldn't work, but my
> preference is that you create a branch with your changes 'squashed' into
> logic chunks for my consumption. Sure, I don't always do the same myself,
> but if I'm submitting my code to someone else, I'll make sure the
> individual commits are readable and functional.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jadonk/bone101/pull/34/files/8d2f6c97f7a6a60660b04bb1ef3c70f3312757b7..876b41a74b2766a1617ef95444801bf337256a1a#r57150448>
>